tfhe/src/shortint/ciphertext/common.rs

```diff
@@ -26,6 +26,10 @@ impl std::error::Error for NotTrivialCiphertextError {}
 pub struct MaxNoiseLevel(usize);
 
 impl MaxNoiseLevel {
+    #[cfg(test)]
+    pub(crate) const MAX: Self = Self(usize::MAX);
```

Comment:
why this config ?

Reply:
Its only used in tests and since its pub(crate) clippy gives an error

Reply:
switch to u64 as well